lib/connection.cpp

```diff
@@ -292,7 +292,7 @@ class Connection::Private {
 
         const auto identityKey = olmAccount->identityKeys().curve25519;
         const auto personalCipherObject =
-            encryptedEvent.ciphertext(identityKey);
+            encryptedEvent.ciphertext(QString::fromUtf8(identityKey));
```

Comment:
The keys are Base64-encoded.
```suggestion
            encryptedEvent.ciphertext(QString::fromLatin1(identityKey));
```